sandbox/apparmor/notify/notify.go

```diff
@@ -0,0 +1,26 @@
+// Package notify implements high-level notify interface to a subset of AppArmor features
+package notify
+
+import (
+	"path/filepath"
+
+	"github.com/snapcore/snapd/dirs"
+	"github.com/snapcore/snapd/osutil"
+)
+
+var NotifyPath string
+
+// PromptingSupportAvailable returns true if NotifyPath exists, indicating that
+// apparmor prompting messages may be received from NotifyPath.
+func PromptingSupportAvailable() bool {
```

Comment:
nitpick: maybe we don't need Prompting in the name

Reply:
Would you suggest `SupportAvailable()`, or something else? I'm not sure if it should be something more descriptive.